assets/js/googlesitekit/datastore/site/info.js

```diff
@@ -161,10 +162,23 @@ export const selectors = {
 	 * @param {Object} state Data store's state.
 	 * @return {(string|undefined)} This site's admin URL.
 	 */
-	getAdminURL: createRegistrySelector( ( select ) => () => {
+	getAdminURL: createRegistrySelector( ( select ) => ( state, page = undefined, args = {} ) => {
 		const { adminURL } = select( STORE_NAME ).getSiteInfo() || {};
 
-		return adminURL;
+		// Prevent a page in args from overriding main page argument
+		const { page: extraPage, ...queryArgs } = args; // eslint-disable-line no-unused-vars
+
+		// Add query arguments to URL if supplied
+		const fullURL = ( adminURL && ( page || Object.keys( queryArgs ).length ) )
```

Comment:
This may not have been formulated in the ACs precisely: The `args` object should only be considered when `page` is present, i.e. calling like `getAdminURL( undefined, { arg1: value1 } )` should have no effect.

Reply:
Ah, got it. I've updated the function and the unit test to cover that.